ts/src/gameClasses/EntityTrack.ts

```diff
@@ -0,0 +1,155 @@
+class EntityTrack {
+	trackEntityById: {[key: string]: IgeEntity};
+	timeStamp: number;
+
+	constructor() {
+		console.log('EntityTrack born');
+
+		this.trackEntityById = {};
+	}
+
+	applyRendererEvents(): void {
```

Comment:
following up to the comment above, since this method is only one line of code, lets remove it and add that line to the end of the constructor